apps/spikeConverter.cpp

```diff
@@ -93,6 +93,13 @@ int main( int argc, char* argv[] )
                   << brion::Version::getString() << std::endl;
         return EXIT_SUCCESS;
     }
+    if( vm["input"].as< std::string >() == vm["output"].as< std::string >( ))
+    {
+        std::cerr << "Cowardly refusing to convert "
+                  << vm["input"].as< std::string >() << " onto itself"
+                  << std::endl;
+        return EXIT_FAILURE;
+    }
```

Comment:
I disagree: by default, it converts a testdata ascii report to a binary format. In particular, using  out.spikes as default for output is what one wants in 80% of the cases, so make simple things simple.

Not sure what you mean with the positional reasoning. Imo that you could do 'spikeConverter in out' was just laziness on the implementers part (yours truly), and the -i/-o is now consistent with the compartmentConverter.